lib/configuration.ts

```diff
@@ -631,7 +631,10 @@ export function loadAutomationConfig(configPath?: string): Configuration | undef
     }
     if (cfgPath) {
         try {
-            const cfg = require(cfgPath).configuration;
+            let cfg = require(cfgPath).configuration;
+            if (cfg instanceof Promise) {
```

Comment:
Is it possible for `require` to return a Promise?

Reply:
Perhaps add a test that demonstrates it?

Reply:
It is not the require that is the Promise. It is the configuration const that’s the Promise.

Reply:
I’ll add a test good point. You can see this in action in the index of the new samples.